mlflow/entities/span.py

```diff
@@ -144,6 +155,16 @@ def outputs(self) -> Any:
         """The output values of the span."""
         return self.get_attribute(SpanAttributeKey.OUTPUTS)
 
+    @property
+    def messages(self) -> str:
+        """The chat messages of the span, if any"""
+        return self.get_attribute(SpanAttributeKey.CHAT_MESSAGES)
```

Comment:
Hmm I'm reluctant to add these properties to the Span object. There is no future limit for what type of spans we will support, and adding specific arguments for each type will make this class dirty. I think asking users to call `get_attribute(spanAttributeKey.CHAT_MESSAGES` is fair as everything used there is public?

Reply:
yeah that's fair, it may get quite crowded